src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/UnstructuredAddressTest.java

```diff
@@ -37,7 +38,17 @@ void testFromStringValidWithMinimumAddressFields() {
         final var address = UnstructuredAddress.fromString("ADR++US:LINE1::::++");
         assertAll(
             () -> assertThat(address.getFormat()).isEqualTo("US"),
-            () -> assertThat(address.getAddressLines()).containsExactly("LINE1", "", "", "", ""),
+            () -> assertThat(address.getAddressLines()).contains(new String[]{"LINE1", "", "", "", ""}),
+            () -> assertThat(address.getPostCode()).isEmpty()
```

Comment:
This test could be renamed `testFromStringValidWithMinimumAddressFieldsAndNoPostcode`

Reply:
Yes, it definitely sounds better! Thanks!
26459dba57b4aff2429f24cae805873125769cf9